Update ARIA: suggestion role incomplete description (#34585)
mdn/content: files/en-us/web/accessibility/aria/roles/suggestion_role/index.md

</div>
```

> Browsers tend to provide a default black strikethrough for deletions, and a black underline for insertions, but you'll probably want to use CSS to customize the visual styling of deletions and insertions.

Reviewer: this should be modified further to emphasize that it's not just that one might want to adjust styling from what the browser provides by default - but that if you're using the ARIA roles, the browser won't even provide those simple defaults and one will need to add their own styling.

```suggestion
Browsers tend to provide a default black strikethrough for deletions, and a black underline for insertions if one is using the HTML elements that implicitly expose these roles. But if one is using explicit ARIA role to modify HTML elements, such as divs, you'll need to use CSS to customize the visual styling for such deletions and insertions.
```

> Browsers tend to provide a default black strikethrough for deletions, and a black underline for insertions if one is using the HTML elements that implicitly expose these roles. But if one is using explicit ARIA role to modify HTML elements, such as divs, you'll need to use CSS to customize the visual styling for such deletions and insertions.

Reviewer 2: ```suggestion
Browsers tend to provide a default black strikethrough for deletions, and a black underline for insertions when using the HTML elements that implicitly expose these roles. But when using explicit ARIA roles to modify HTML elements, such as divs, you'll need to use CSS to customize the visual styling for such deletions and insertions.
```
